### ISSUE TITLE
Max-age header is ignored

### ISSUE THREAD

**Issue by jindrazak** [llfbandit/dio_cache_interceptor#14]
In my API I store almost immutable data and I want them to be cached. However, I am not easily able to add ETag header or last-modified header. Therefore I use following in my responses:

```http
cache-control: max-age=1800,public
```

According to MDN in terms of caching, it should behave as static assets and **should be cached**.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#caching_static_assets

But the `dio_cache_interceptor` (when using `CachePolicy.request` policy) considers cacheable only those requests with `ETag` or `last-modified` headers present. 
https://github.com/llfbandit/dio_cache_interceptor/blob/master/lib/src/dio_cache_interceptor.dart#L145

I would find it very helpful if the requests with max-age would be cached as well.

**Comment by llfbandit** [comment]
Indeed, `max-age` is not enough to trigger caching.
With `max-age` only, there is no way to properly validate cache against origin server.

This could be done with request date to generate `If-Modified-Since` header but this is not compliant with current techniques and I guess you won't be able to return 304 since you don't generate `Last-Modified` header.

Also, if `max-age` is set statically, I could suggest you to to use forceCache options with maxStale since there is no cache on origin server.

**Comment by jindrazak** [comment]
Yes, I solved it with `forceCache` which works fine for my case because all the response headers contain `max-age` header. However, I guess that this approach wouldn't work if some of the responses contained some different cache-related headers.

At least I would propose adding some info about the relevant HTTP headers to the documentation of this package because I had to go to the source code and search for which headers are used and which aren't.

Also, thank you for the quick response! 👍
